src/handlebar-helpers.js

```diff
@@ -57,7 +58,11 @@ const helpers = {
       return arr.includes(valueToFind, fromIndex);
     }
     return false;
-  }  
+  },
+  sha256: function(data) {
+    const hash = crypto.createHash('sha256').update(data).digest('hex');
```

Comment:
please validate data is a string.

crypto allows data to be one of type string or an instance of Buffer, TypedArray, or DataView, but unless you need those others, we can just do string for now.

if its not string, maybe just return empty string? or the original data. i'll leave that decision up to you.